opus/application/static_media/js/browse.js

```diff
@@ -561,51 +557,45 @@ var o_browse = {
         o_browse.loadData(opus.prefs.view, 1);
     },
 
-    // check if we need infiniteScroll to load next page when there is no more prefected data
-    loadNextPageIfNeeded: function(opusId) {
+    loadPageIfNeeded: function(direction, opusId) {
         // opusId will be empty at the end of the observations, so just return out.
         if (opusId === "") {
             return;
         }
         let startObsLabel = o_browse.getStartObsLabel();
         let tab = opus.getViewTab();
         let contentsView = o_browse.getScrollContainerClass();
+        o_browse.currentOpusId = opusId;
```

Comment:
Also wondering about this name. I assume this is where the binoculars are pointed? That's not obvious from the name. current opus id is pretty generic. How about something like "metadataDetailOpusId"?